src/context/AppProvider/AppContext.tsx

```diff
@@ -114,31 +118,55 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
   useEffect(() => {
     if (!wallet) return
     ;(async () => {
-      try {
-        const accountNumber = 0
-        const chainIds = supportedChains
-        const accountMetadataByAccountId = await deriveAccountIdsAndMetadata({
-          accountNumber,
-          chainIds,
-          wallet,
+      let chainIds = Array.from(supportedChains).filter(chainId =>
```

Comment:
`const`? lodash's `pull` can still mutate the array - only the array's memory address is constant.